sdk/js/src/solana/wormhole/accounts/config.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  Connection,
+  PublicKey,
+  Commitment,
+  PublicKeyInitData,
+} from "@solana/web3.js";
+import { deriveAddress, getAccountData } from "../../utils";
+
+export function deriveWormholeBridgeDataKey(
+  wormholeProgramId: PublicKeyInitData
+): PublicKey {
+  return deriveAddress([Buffer.from("Bridge")], wormholeProgramId);
+}
+
+export async function getWormholeBridgeData(
+  connection: Connection,
+  wormholeProgramId: PublicKeyInitData,
+  commitment?: Commitment
+): Promise<BridgeData> {
+  return connection
+    .getAccountInfo(deriveWormholeBridgeDataKey(wormholeProgramId), commitment)
+    .then((info) => BridgeData.deserialize(getAccountData(info)));
+}
+
+export class BridgeConfig {
+  // pub struct BridgeConfig {
```

Comment:
do we want to keep this commented out code?

Reply:
Ah, these were just referencing the structs of accounts if anyone wanted to look at the source. But I can take these out